src/Compilers/Core/Portable/MetadataReader/PEModule.cs

```diff
@@ -46,6 +46,9 @@ internal sealed class PEModule : IDisposable
 
         private ImmutableArray<AssemblyIdentity> _lazyAssemblyReferences;
 
+        private static readonly Dictionary<string, (int, int)> s_sharedEmptyForwardedTypes = new Dictionary<string, (int, int)>();
+        private static readonly Dictionary<string, (int, int)> s_sharedEmptyCaseInsensitiveForwardedTypes = new Dictionary<string, (int, int)>();
```

Comment:
>s_sharedEmptyCaseInsensitiveForwardedTypes [](http://example.com/codeflow?start=63&length=42)

Can we share ```s_sharedEmptyForwardedTypes``` field for this purpose? It doesn't look like the instances have different characteristics.